loader/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/loader/Loader.scala

```diff
@@ -71,8 +70,12 @@ object Loader {
       .flatMap {
         case (Right(row), ctx) =>
           ctx.output(ObservedTypesOutput, row.inventory)
-          val latency = (Instant.now().getMillis - row.collectorTstamp.getMillis) / 1000
-          latencyDistribution.update(latency)
+          val diff = Instant.now().getMillis - row.collectorTstamp.getMillis
```

Comment:
More of a general question on how we manage the kind of metrics, as it seems like we rely heavily on time syncing across machines to calculate that. 
For a relative metric it isn't absolutely necessary to have a very precise values but reliance on distributed time syncing can lead to unreliable values ie. through time drift or even complete clock back. Brings the point of general approach to time and just a reminder so we keep that in mind.